example/rest/orders-walkthrough/app/config.go

```diff
@@ -0,0 +1,16 @@
+package app
+
+import (
+	"github.com/z5labs/humus/rest"
+)
+
+// Config defines the application configuration.
+type Config struct {
```

Comment:
This should be in `app.go` just like the other examples. Any related walkthrough docs that reference this file should be updated too.